app/src/main/java/appricottsoftware/clarity/fragments/ChannelFragment.java

```diff
@@ -167,8 +156,12 @@ public void onClick(View v) {
                         @Override
                         public void onClick(View view) {
 
-                            rListItemsSearch = new ArrayList<>();
-                            rAdapterSearch = new RecyclerAdapter(rListItemsSearch, getContext(), false);
+//                            rListItemsSearch = new ArrayList<>();
+//                            rAdapterSearch = new RecyclerAdapter(rListItemsSearch, getContext(), false);
+//                            channelRecycler.setAdapter(rAdapterSearch);
+
+                            searchChannels = new ArrayList<>();
+                            rAdapterSearch = new ChannelsAdapter(searchChannels, getContext(), false);
                             channelRecycler.setAdapter(rAdapterSearch);
```

Comment:
Could we get rid of these commented code? If not, please write a comment describing why the comments are necessary

Reply:
@mschaffrath I'm removing old code.